pwiz_tools/Skyline/Controls/Startup/ActionTutorial.cs

```diff
@@ -234,7 +241,20 @@ public void ExtractTutorial(ILongWaitBroker waitBroker)
                         MessageDlg.ShowWithException(Program.MainWindow, message, e);
                     }
                 }));
-               
+
+                // Make it convenient for user to locate tutorial files if we haven't already opened anything
+                if (!hasSkylineFile && !string.IsNullOrEmpty(ExtractPath))
+                {
+                    Directory.SetCurrentDirectory(ExtractPath);
+                    Settings.Default.LibraryDirectory =

```

Comment:
Setting all of these directories at the same time looks like neat functionality. I wonder if we should make it a method on "DirectoryEx".

Reply:
DirectoryEx.SetSkylineWorkingDirectories() ?